src/app/search-result/search-result.component.ts

```diff
@@ -33,16 +33,31 @@ export class SearchResultComponent implements OnInit {
 
           this.requestRepoService.fetchRepos(searchTerm).subscribe(_ => {
             this.isFetching = false;
-            this.searchResults = this.searchResultService.getSearchResult();
+            this.searchResults = this.searchResultService.getSearchResult().slice();
             if (this.searchResults.length === 0) {
               this.errorMsg = 'No Project Related to ' + searchTerm + ' ......';
+            } else {
+              this.addPlatformUrl();
             }
           }, error => {
             this.errorMsg = error.error.message;
           });
       });
   }
 
+  addPlatformUrl() {
```

Comment:
should we keep it consistent by using the logo from font-awesome? you can refer to [here](https://github.com/Kaihchen1230/senior-design-frontend/blob/6f934bbf3c2010201f25b7d3ad51c474187178ad/src/app/detail-content/similar-repos/similar-repos.component.ts#L4).